R/tokens.R

```diff
@@ -262,6 +262,13 @@ tokens.corpus <- function(x,
                            fasterword = tokenize_fasterword,
                            fastestword = tokenize_fastestword)
     
+    
+    if (what == "word2") {
+        x <- preserve_special2(x, split_hyphens = split_hyphens, 
+                               split_tags = FALSE, verbose = verbose)
```

Comment:
I had to add `split_tags = FALSE` because otherwise it was always letting tags be split.